### PR TITLE
update docs landing

### PR DESCRIPTION
**Pull request by alex-fusionauth**
refers to https://github.com/FusionAuth/fusionauth-issues/issues/2570

---
- To see the specific tasks where the Asana app for GitHub is being used, see below:
  - https://app.asana.com/0/0/1206300632702801

### CODE REVIEW

**Comment by mooreds** [comment]
@alex-fusionauth I think we weren't clear with the instructions, sorry. This doesn't match what I had in mind at all.

I created a couple of low fidelity mockups.

Here's where we add SDKs and APIs sections to the 'other resources' section. We should also add the 'reference' section here.

Screenshot 2024-02-15 at 2 26 30 PM

Here's where we add a new section called 'documentation categories' which has all 5 documentation categories (though my mockup only shows four).

* get started
* lifecycle
* customize
* extend
* operate

Screenshot 2024-02-15 at 2 26 37 PM

The mockups have the headings, but not the text. I can provide the descriptive text for each section. The links should take you to the same place that the headings in the nav dropdown do.

Please let me know if this makes sense or if we should do a quick sync chat.

**Review by mooreds — CHANGES_REQUESTED** [review]
See comment about sections.

File: astro/src/pages/docs/index.astro [hunk 1]
```diff
@@ -107,6 +107,12 @@ const otherResources = [
 		"icon": "/img/icons/release-notes.svg",
 		"href": "/docs/release-notes/"
 	},
+	{
+		"title": "Reference",
```
Comment (mooreds) [inline astro/src/pages/docs/index.astro:111]: Looks like `Reference` is in this section twice.

File: astro/src/pages/docs/index.astro [hunk 2]
```diff
@@ -124,8 +130,60 @@ const otherResources = [
 		"description": "Best practices, advice, and real world use cases to help you hone your expertise in all things auth.",
 		"icon": "/img/icons/articles.svg",
 		"href": "/articles/"
+	},
+	{
+		"title": "SDKs",
+		"description": "An opinionated set of higher-level constructs designed to simplify the integration of FusionAuth authentication workflows into applications.",
```
Comment (mooreds) [inline astro/src/pages/docs/index.astro:136]:
```suggestion
		"description": "Learn how to integrate and manage FusionAuth using your favorite programming language. You can choose either our client libraries (a thin, low-level wrapper over our APIs) or our SDKs (an opinionated set of higher-level constructs designed to simplify the integration of FusionAuth authentication workflows into applications).",
```

File: astro/src/pages/docs/index.astro [hunk 3]
```diff
@@ -124,8 +130,60 @@ const otherResources = [
...
+		"icon": "/img/icons/sdk.svg",
+		"href": "/docs/sdks/"
+	},
+	{
+		"title": "APIs",
+		"description": "A comprehensive set of RESTful APIs that enable you to seamlessly integrate login, registration, and advanced user management features into your application.",
```
Comment (mooreds) [inline astro/src/pages/docs/index.astro:142]:
```suggestion
		"description": "A comprehensive set of RESTful APIs that enable you to integrate login, registration, and advanced user management features into your application. You can also build your own workflows and completely manage your FusionAuth instance using these APIs.",
```

File: astro/src/pages/docs/index.astro [hunk 4]
```diff
@@ -124,8 +130,60 @@ const otherResources = [
...
+		"description": "Provides detailed information on various aspects of the system, including configuration options, data types, cookies set by the Hosted Login pages, and supported password hashing algorithms.",
+		"icon": "/img/icons/reference.svg",
+		"href": "/docs/reference/"
+	}
+];
+
+const docCategories = [
+	{
+		"title": "Get Started",
+		"description": "Overview of how to use our quick start guides, installation options and how to call our APIs.",
```
Comment (mooreds) [inline astro/src/pages/docs/index.astro:157]:
```suggestion
		"description": "How to install and run FusionAuth, as well as documentation about core concepts such as tenants, users and applications.",
```

File: astro/src/pages/docs/index.astro [hunk 5]
```diff
@@ -124,8 +130,60 @@ const otherResources = [
...
+		"icon": "/img/icons/add-mfa.svg",
+		"href": "/docs/get-started/"
+	},
+	{
+		"title": "Lifecycle",
+		"description": "The FusionAuth lifecycle encompasses various stages and processes involved in managing user identities and access within the FusionAuth system",
```
Comment (mooreds) [inline astro/src/pages/docs/index.astro:163]:
```suggestion
		"description": "Covers registration/sign up, login/sign in, user management and migration.",
```

File: astro/src/pages/docs/index.astro [hunk 6]
```diff
@@ -124,8 +130,60 @@ const otherResources = [
...
+		"icon": "/img/icons/add-mfa.svg",
+		"href": "/docs/lifecycle/"
+	},
+	{
+		"title": "Customize",
+		"description": "FusionAuth provides extensive customization options for enhancing the user experience and integrating your application seamlessly",
```
Comment (mooreds) [inline astro/src/pages/docs/index.astro:169]:
```suggestion
		"description": "How to customize the look and feel of pages and emails your users will see when they are logging in or registering.",
```

File: astro/src/pages/docs/index.astro [hunk 7]
```diff
@@ -124,8 +130,60 @@ const otherResources = [
...
+		"icon": "/img/icons/add-mfa.svg",
+		"href": "/docs/customize/"
+	},
+	{
+		"title": "Extend",
+		"description": "FusionAuth provides a mechanism to extend its functionality through plugins. This extension capability is designed to enhance the system's performance by loading plugins at startup rather than at runtime.",
```
Comment (mooreds) [inline astro/src/pages/docs/index.astro:175]:
```suggestion
		"description": "Covers how to extend FusionAuth, including run time modifications of tokens, webhooks and events, and example applications.",
```

File: astro/src/pages/docs/index.astro [hunk 8]
```diff
@@ -124,8 +130,60 @@ const otherResources = [
...
+		"icon": "/img/icons/add-mfa.svg",
+		"href": "/docs/extend/"
+	},
+	{
+		"title": "Operate",
+		"description": "A collection of documentation and resources aimed at helping users manage, secure, and monitor their FusionAuth installation effectively.",
```
Comment (mooreds) [inline astro/src/pages/docs/index.astro:181]:
```suggestion
		"description": "How to operate FusionAuth, including advanced deployment configurations, Terraform and configuration management options, troubleshooting, and security and monitoring information.",
```

**Review by mooreds — CHANGES_REQUESTED** [review]
Reviewed text, added suggestions.

File: astro/public/img/icons/ruby.svg
Comment (mooreds) [inline astro/public/img/icons/ruby.svg:1]: do we need this any more?

File: astro/public/img/icons/typescript.svg
Comment (mooreds) [inline astro/public/img/icons/typescript.svg:1]: do we need this?

File: astro/src/pages/docs/index.astro (hunk 2 above)
Comment (mooreds) [reply astro/src/pages/docs/index.astro:130]: Did you mean to resolve this and ignore my text suggestion?

File: astro/public/img/icons/openapi.svg
Comment (mooreds) [inline astro/public/img/icons/openapi.svg:1]: do we need this?

File: astro/public/img/icons/java.svg
Comment (mooreds) [inline astro/public/img/icons/java.svg:1]: do we need this?

File: astro/public/img/icons/dart.svg
Comment (mooreds) [inline astro/public/img/icons/dart.svg:1]: do we need this?

**Review by mooreds — CHANGES_REQUESTED** [review]

**Review by alex-fusionauth — COMMENTED** [review]
updated on last review

**Review by mooreds — APPROVED** [review]

**Pull request merged**